ConsoleApplication/Program.cs

```diff
@@ -44,7 +46,9 @@ public static async Task Main (string[] args)
       var solution = await LoadSolutionSpace (solutionPath);
       var project = LoadProject (solution, projectName);
 
-      foreach (var document in project.Documents)
+      var documents = await new InheritanceProjectConverter().Convert (project.Documents);

```

Comment:
Shouldn't this be the other way around? As I understood it, the `InheritanceProjectConverter` "relies" on changes done before by the other converters?